shared/events.d.ts

```diff
@@ -4,88 +4,88 @@
-	'HideMainMenu':						() => void,
-	'HudProcessInput':					() => void,
-	'HudThink':							() => void,
-	'ShowIntroMovie':					() => void,
-	'HideIntroMovie':					() => void,
-	'DemoPlaybackControl': 				(str: string, flt: float) => void,
-	'Drawer_ExtendAndNavigateToTab':	(tabid: string) => void,
-	'Drawer_NavigateToTab': 			(tabid: string) => void,
-	'Drawer_UpdateLobbyButton':			(imgsrc: string, playercount: unknown) => void,
-	'GameEventFired':					(event: string) => void,
-	'HideContentPanel':					() => void,
-	'LayoutReloaded':					() => void,
-	'MainMenuTabHidden':				(tabid: string) => void,
-	'MainMenuTabShown':					(tabid: string) => void,
-	'PageDown':							() => void,
-	'PageLeft':							() => void,
-	'PageRight':						() => void,
-	'PageUp':							() => void,
-	'PanoramaGameTimeJumpEvent':		(time: duration) => void,
-	'ReloadBackground':					() => void,
-	'ScrollDown':						() => void,
-	'ScrollLeft':						() => void,
-	'ScrollRight':						() => void,
-	'ScrollUp':							() => void,
-	'SettingsNavigateToPanel':			(category: string, panel: Panel) => void,
-	'ShowCenterPrintText':				(message: string, priority: unknown) => void,
-	'ShowContentPanel':					() => void,
-	'ShowVoteContextMenu':				() => void,
-	'StaticHudMenu_EntrySelected':		(panel: Panel) => void,
-	'UnloadLoadingScreenAndReinit':		() => void,
-	'AchievementInfoLoaded':			() => void,
-	'AchievementEarned':				(player_index: number, achievement_index: number) => void,
+	AsyncEvent: (delay: duration, eventToFire: string) => void;
+	MainMenuResumeGame: () => void;
+	MainMenuPauseGame: () => void;
+	ShowPauseMenu: () => void;
+	HidePauseMenu: () => void;
+	ShowMainMenu: () => void;
+	HideMainMenu: () => void;
+	HudProcessInput: () => void;
+	HudThink: () => void;
+	ShowIntroMovie: () => void;
+	HideIntroMovie: () => void;
+	DemoPlaybackControl: (str: string, flt: float) => void;
+	Drawer_ExtendAndNavigateToTab: (tabid: string) => void;
+	Drawer_NavigateToTab: (tabid: string) => void;
+	Drawer_UpdateLobbyButton: (imgsrc: string, playercount: unknown) => void;
+	GameEventFired: (event: string) => void;
+	HideContentPanel: () => void;
+	LayoutReloaded: () => void;
+	MainMenuTabHidden: (tabid: string) => void;
+	MainMenuTabShown: (tabid: string) => void;
+	PageDown: () => void;
+	PageLeft: () => void;
+	PageRight: () => void;
+	PageUp: () => void;
+	PanoramaGameTimeJumpEvent: (time: duration) => void;
+	ReloadBackground: () => void;
+	ScrollDown: () => void;
+	ScrollLeft: () => void;
+	ScrollRight: () => void;
+	ScrollUp: () => void;
+	SettingsNavigateToPanel: (category: string, panel: Panel) => void;
+	ShowCenterPrintText: (message: string, priority: unknown) => void;
+	ShowContentPanel: () => void;
+	ShowVoteContextMenu: () => void;
+	StaticHudMenu_EntrySelected: (panel: Panel) => void;
+	UnloadLoadingScreenAndReinit: () => void;
+	AchievementInfoLoaded: () => void;
+	AchievementEarned: (player_index: number, achievement_index: number) => void;
```

Comment:
is there anything we can do so it doesn't totally fuck up the formatting of this file?

Reply:
No, prettier will always get rid of indentation like this, personally I prefer it prettier's way, find it easier to read and edit, if you really can't stand it we wouldn't be able to use prettier (happy to discuss that).